twittermap/public/javascripts/map/controllers.js

```diff
@@ -671,7 +671,7 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common'])
           if($scope.doSentiment) {
             $scope.infoPromp = "Score";  // change the info promp
```

Comment:
Since you are using config to set the prompt, I think maybe you can remove this line.